src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PayPalMapper.php

```diff
@@ -114,19 +141,33 @@ protected function mapAddressFromQuoteToComputopPayPalPayment(
      *
      * @return array
      */
-    protected function getDataSubArray(ComputopPayPalPaymentTransfer $computopPayPalPaymentTransfer)
+    protected function getDataSubArray(ComputopPayPalPaymentTransfer $computopPayPalPaymentTransfer): array
     {
         $dataSubArray[ComputopApiConfig::TRANS_ID] = $computopPayPalPaymentTransfer->getTransId();
         $dataSubArray[ComputopApiConfig::AMOUNT] = $computopPayPalPaymentTransfer->getAmount();
         $dataSubArray[ComputopApiConfig::CURRENCY] = $computopPayPalPaymentTransfer->getCurrency();
-        $dataSubArray[ComputopApiConfig::URL_SUCCESS] = $computopPayPalPaymentTransfer->getUrlSuccess();
-        $dataSubArray[ComputopApiConfig::URL_NOTIFY] = $computopPayPalPaymentTransfer->getUrlNotify();
-        $dataSubArray[ComputopApiConfig::URL_FAILURE] = $computopPayPalPaymentTransfer->getUrlFailure();
         $dataSubArray[ComputopApiConfig::CAPTURE] = $computopPayPalPaymentTransfer->getCapture();
-        $dataSubArray[ComputopApiConfig::RESPONSE] = $computopPayPalPaymentTransfer->getResponse();
-        $dataSubArray[ComputopApiConfig::MAC] = $computopPayPalPaymentTransfer->getMac();
         $dataSubArray[ComputopApiConfig::TX_TYPE] = $computopPayPalPaymentTransfer->getTxType();
         $dataSubArray[ComputopApiConfig::ORDER_DESC] = $computopPayPalPaymentTransfer->getOrderDesc();
+
+        $orderDescriptions = $computopPayPalPaymentTransfer->getOrderDescriptions();
+
+        $orderDescriptionsCount = count($orderDescriptions);
+        for ($key = static::PAYPAL_ITEM_DESCRIPTION_OFFSET; $key < $orderDescriptionsCount; $key++) {
+            $dataSubArray[$this->getOrderItemDescriptionKey($key)] = $orderDescriptions[$key];
```

Comment:
* I see an issue here, you have to add one more variable to store an increment for reading the description in your case you will lose 2 records.